test/wowman/curseforge_api_test.clj

```diff
@@ -75,7 +75,89 @@
           fixture {:latestFiles latest-files}
           expected {"wow_retail" [{:gameVersionFlavor "wow_retail", :fileDate "2001-01-01T00:00:00.000Z", :releaseType stable, :exposeAsAlternative nil}]
                     "wow_classic" [{:gameVersionFlavor "wow_classic", :fileDate "2001-01-01T00:00:00.000Z", :releaseType stable, :exposeAsAlternative nil}]}]
-      (is (= expected (curseforge-api/latest-versions-by-game-track fixture))))))
+      (is (= expected (curseforge-api/latest-versions-by-gameVersionFlavor fixture))))))
+
+(deftest latest-versions-by-gameVersion
+  (testing "releases supporting multiple game tracks are expanded into multiple releases"
+    (let [stable 1
+          latest-files [{:gameVersionFlavor "wow_retail", :gameVersion ["1.13.1" "8.2.5"]
+                         :fileDate "2001-01-03T00:00:00.000Z", :releaseType stable, :exposeAsAlternative nil}]
+          fixture {:latestFiles latest-files}
+
+          expected {"wow_retail" [{:gameVersionFlavor "wow_retail", :gameVersion ["8.2.5"]
+                                   :fileDate "2001-01-03T00:00:00.000Z", :releaseType stable, :exposeAsAlternative nil}]
+
+                    "wow_classic" [{:gameVersionFlavor "wow_classic", :gameVersion ["1.13.1"]
+                                    :fileDate "2001-01-03T00:00:00.000Z", :releaseType stable, :exposeAsAlternative nil}]}]
+      (is (= expected (curseforge-api/latest-versions-by-gameVersion fixture)))))
+
+  (testing "previous releases for different game tracks not supported in later releases are available"
```

Comment:
"previous releases with multiple game tracks are still available even when dropped in the latest release"

or something